### ISSUE TITLE
Screen Reader: Settings-Troubleshoot: Submit a bug button not focusable.

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76070]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** undefined
**Reproducible in staging?:** Yes
**Reproducible in production?:** undefined
**If this was caught during regression testing, add the test name, ID and link from BrowserStack:** undefined
**Email or phone of affected tester (no customers):** undefined
**Issue reported by:** Applause Internal Team
**Device used:** undefined
**App Component:** undefined

## Action Performed:
Prerequisite:
User is logged in

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/troubleshoot
2. Attempt to navigate using TAB to the 'Submit a bug' button within the 'Troubleshoot' informational paragraph
3. Observe the element is not focusable via TAB key.

## Expected Result:
The 'Submit a bug' element should be focused with Tab key and announced with a actionable role'.

## Actual Result:
The 'Submit a bug' button, located in the informational paragraph under 'Troubleshoot' is not focusable with Tab key. During navigation this element is skipped. Hence, 'Submit a bug' button and its associated function will not be available for the keyboard users. 

HTML:
<span class="css-1jxf684 r-1loqt21" style="color: rgb(1, 100, 191); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; text-decoration-color: rgb(1, 100, 191); text-decoration-line: none; line-height: 20px;">submit a bug</span>

## Workaround:
Unknown

## Platforms:
- [ ] Android: App
- [ ] Android: mWeb Chrome
- [ ] iOS: App
- [ ] iOS: mWeb Safari
- [ ] iOS: mWeb Chrome
- [x] Windows: Chrome
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

## Area issue was found in:
Settings-Troubleshoot

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Make the control 'Submit a bug' tab focusable by provide an actionable to the control using native elements such button.

## Screenshots/Videos
<img width="1917" height="1003" alt="Image" src="https://github.com/user-attachments/assets/7a358875-3a2d-43fa-8995-7b220edb3e29" />

https://github.com/user-attachments/assets/42c99174-327d-40bd-8744-498846471e18

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
